lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -77,41 +77,81 @@ namespace Intl
         void Cleanup() override
         {
             delete intlObject;
-            intlObject = nullptr;
```

Comment:
I don't necessarily agree with removing this. It's a reflex for me to set a deleted pointer to nullptr to prevent double free if the destructor is called at some point later than expected. Doing `delete this` in the next line pretty much covers that, but still... (I'm imagining the recycler calls the destructor on the memory which used to point to this instance at some point later and then the pointer at `intlObject` is freed again but that might be overly paranoid.

Reply:
I can't think of a scenario where that would ever happen. The only RecyclableObject is AutoIcuJsObject here -- even if the recycler runs it's (default) destructor in addition to calling Finalize, the default destructor still doesn't delete member pointers, since that memory may still be used by a different object. So I can't think of a way that `AutoIcuJsObject<IPlatformAgnosticResource>::resource` would ever be deleted by anything but itself (through `delete this;`)

Reply:
Yeah you're probably right. And anyway if this code might be going away, don't worry to much about it until we're in a more final state.